app/frontend/src/utils/util.js

```diff
@@ -0,0 +1,15 @@
+
+import { KcClientStatus } from '@/utils/constants';
+
+/**
+ * @function buildClientSpan
+ * Returns the appropriate DOM for client Available or Not available (wording and CSS)
+ * @param {string} envLabel Which environment (DEV, TEST, PROD)
+ * @param {boolean} status Whether the service client is there
+ * @returns {string} HTML
+ */
+export function buildClientStatusSpan(envLabel, status) {
+  const cls = status ? 'green--text' : '';
+  const txt = status ? KcClientStatus.AVAILABLE : KcClientStatus.NOT;
+  return `${envLabel}: <span class="${cls}">${txt}</span>`;
+}
```

Comment:
Not quite to convinced this should be abstracted out as a common utils function - most of the logic is basically checking if the text is green, and if it is, saying it's also available. This may be business logic better suited to live in the component it came from, unless this is being utilized in multiple components.

Reply:
Yeah it's used in 2 different components so I refactored it out. I'm not crazy about View building in a utils class...
... oh wait... it should just be its own small component then with props passed in.
If you think that makes sense let me know. I'll do it in my next PR.

Reply:
This would be one of those fantastic cases where we can use a "functional SFC" and pass in via prop. I think that may make more sense - if nothing else that should be something we can investigate. Here's a pretty good article on it and it has an example SFC that this could match with:
https://dev.to/fdietz/vue-js-functional-components-2495

Reply:
Will read up on that for next PR.